### ISSUE TITLE
[Intermittent] Websites can not be scrolled if navigated to from an allowed pop-up blocked page

### ISSUE THREAD

**Issue by heres-maria** [MozillaReality/FirefoxReality#2789]
**[Note]:**
- This issue was not reproducible in 100% of cases on www.popuptest.com. The Mouseover Popup seems to be the one for which it is the most reproducible.

**[Affected Versions]:**
- Firefox Reality 9 RC 1 (33f6c4d8)
…
- Firefox Reality version 9-40450030 (a81f0f08)

**[Affected Platforms]:**
- Oculus Quest - Android 7.1.1
- Oculus Go - Android 7.1.2
- HTC Vive Focus Plus - Android 7.1.2
- HTC Vive Focus - Android 7.1.2
- Pico Neo 2 - Android - 8.1.0

**[Prerequisites]:**
- The headset is turned on, the browser is installed and open.
- The "Block Pop-Up Windows" option is disabled.

**[Steps to reproduce]:**                                        
1. Open [a page that uses pop-ups](http://www.popuptest.com/popuptest4.html).
2. Trigger the pop-up or wait for it to be triggered.
3. In the same tab as the pop-up, navigate to a website.
4. Try to scroll the page and observe the behavior.

**[Expected results]:**
- The page can be scrolled.

**[Actual results]:**
- The page can not be scrolled.

**[Additional Notes]:**
- Probably related to issue  #2684.
- The issue is also reproducible if the "Block Pop-Up Windows" option is enabled and a pop-up technique is allowed.
- Attached is a [recording](https://drive.google.com/file/d/1eDZoy9K6pQvyfOnvidrsI-Al8FB8GBWg/view?usp=sharing) of the issue.

**Comment by keianhzo** [comment]
Looks like a GV regression: https://bugzilla.mozilla.org/show_bug.cgi?id=1616063

**Comment by heres-maria** [comment]
I have verified this issue and it is no longer reproducible with the latest Firefox Reality RC2 (37ac4cbd) build.
As long as the "Multiprocess" option remains disabled, the issue is not reproducible.
It is reproducible if the user turns "Multiprocess" on.
Tested on Oculus Go (Android 7.1.2), Oculus Quest (Android 7.1.1), HTC Vive Focus Plus (Android 7.1.2), HTC Vive Focus (Android 7.1.2) and Pico Neo 2 (Android 8.1.0).
